### DISCUSSION TITLE
❌invalid : In the Background Type section, separate the color and image from each other.

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21724] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team.

Add a container. In the Style > Background Type tab, the color and image are in the classic section. 

![a](https://user-images.githubusercontent.com/88034902/226766989-a8a5e0a2-1a87-43a6-a048-c015290fa271.png)

**But why color and image are in a place called classic? Why is this part called classic?**
I think it would be much better if the color and image were displayed in two separate sections.

### Describe the solution you'd like

In the Background Type section, separate the color and image from each other. Exactly what you see in the picture below :

![elementor](https://user-images.githubusercontent.com/88034902/226766639-0e05734d-feb8-46a4-b0e0-0fa4dac1f726.jpg)

**Comment by m1ga** [comment]
no, then you couldn't upload a PNG file and set a background color.
Having color + background-image in one section is very useful. If the icon should be a brush or something else is debatable. But don't split image and color!

**Reply by Michaelgimii** [reply]
It is reasonable 👍
